libbeat/publisher/publish.go

```diff
@@ -176,6 +176,12 @@ func (publisher *PublisherType) Init(
 	publisher.disabled = *publishDisabled
 	if publisher.disabled {
 		logp.Info("Dry run mode. All output types except the file based one are disabled.")
+	} else if len(configs) == 0 {
+		logp.Info("No output set. Fall back to standard output elasticsearch on localhost:9200")
```

Comment:
@urso You mean instead of logging that ES is not available, that we shut down with an error in case ES is not running? I would keep it the way it is.

@andrewkroh Probably it makes sense to split up this PR into two parts as the second parts is definitively more controversial :-)